app/src/main/java/com/shagalalab/constitution/chapter/ChapterViewModel.kt

```diff
@@ -0,0 +1,19 @@
+package com.shagalalab.constitution.chapter
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.shagalalab.constitution.data.dao.ChapterDao
+import com.shagalalab.constitution.data.models.ChapterModel
+import java.util.concurrent.Executors
+
+class ChapterViewModel(private val chapterDao: ChapterDao) : ViewModel() {
+
+    private var chapterList: MutableLiveData<List<ChapterModel>> = MutableLiveData()
+    var chapterListToObserveOutside: MutableLiveData<List<ChapterModel>> = MutableLiveData()
```

Comment:
This is not exactly what I asked. Now, what's the difference between `chapterList` and `chapterListToObserveOutside`? They are the same, both returning `MutableLiveData<List<ChapterModel>>`. You did correctly first part, changing `chapterList` to private. However, I asked `chapterListToObserveOutside` (this is a horrible name, you need to change it, you can change `chapterList` to `chapterListLiveData`, and this one to `chapterList`) to return LiveData. The only thing you need to was to create a variable (which you did, just change it to `val`, just remember always create variables `val` unless you need to change it). 

<img width="378" alt="Screen Shot 2019-10-30 at 08 01 24" src="https://user-images.githubusercontent.com/8268042/67808821-8230b700-faeb-11e9-90b1-8846a76b3736.png">
What we can understand from `MutableLiveData` class declaration? That it is extending `LiveData`. That means we can assign `MutableLiveData` variables to `LiveData` ones, because `MutableLiveData` already implements everything that `LiveData` requires.